resources/language/German/strings.po

```diff
@@ -474,7 +474,7 @@ msgstr "Empfange Wiedergabefortschritt der Episoden von Trakt"
 
 msgctxt "#32120"
 msgid "Parsing episode playback progress %i of %i from Trakt"
-msgstr "Verarbeite Wiedergabefortschritt der Episoden %i of %i von Trakt"
+msgstr "Verarbeite Wiedergabefortschritt der Episoden %i von %i von Trakt"
```

Comment:
Yeah, also thought about that. But, I'm german and it's still pretty impossible for me to guess the real context in which this statement is going to show up (when I only look at the german statement). Or just leave Trakt. Do we really need it here?